examples/autokey-simple/main.tf

```diff
@@ -0,0 +1,22 @@
+/**
```

Comment:
do we need a README.md file for each example?

Reply:
Good point, it can be helpful. But for now, none of the existing examples in our `examples/` folder has it.
Is it OK to create a dedicated PR to address that, after we merge this one? I can add it all at once then.

Reply:
it's fine to track for later into backlog; to be consistent and the PR size.